Changelog.md

```diff
@@ -1,3 +1,42 @@
+### 0.10.1-alpha.1 (2022-06-22)
+
+Features:
+
+* Supported WASI-NN plug-in with OpenVINO backend on Ubuntu 20.04.
+* Supported the WASM `threads` proposal.
+  * Added the `WasmEdge_Proposal_Threads` for the configuration in WasmEdge C API.
+  * Users can use the `--enable-threads` to enable the proposal in `wasmedge` and `wasmedgec` tools.
+
+Fixed issues:
+
+* Fixed the WasmEdge C API static library on MacOS with LLVM 14.
+  * The WasmEdge C API static library is in experimental and not guaranteed. The shared library is recommended.
+
+Known issues:
+
+* Universal WASM format failed on MacOS platforms.
+  * In current status, the universal WASM format output of the AOT compiler on MacOS platforms will cause bus error when execution.
+  * We are trying to fix this issue. For working around, please use the shared library format output of the AOT mode.
+  * Developers can specify the extension name as `.dylib` on MacOS, `.so` on Linux, and `.dll` on Windows for the shared library format output of the AOT compiler.
+* Plug-in linking on MacOS platforms.
+  * The plug-in on MacOS platforms will cause symbol not found when dynamic linking.
+  * We are trying to fix this issue. For working around, please implement the host modules instead of plug-ins.
```

Comment:
Please also add #1559. If someone has the same problem `MSVCP140.dll is missing`, then this issue can be a workaround.